src/main/java/rcms/utilities/daqexpert/persistence/LogicModuleRegistry.java

```diff
@@ -89,6 +90,8 @@
 
 	FEROLFifoStuck		   		(new FEROLFifoStuck(),		       ConditionGroup.OTHER,                 "",                                                  500,  10500),
 
+	RuFailed                (new RuFailed(),                 ConditionGroup.OTHER,                 "",                                                  900,   9500),
```

Comment:
the reasoning behind the high run order was that -- similar to the `UnidentifiedFailure` module, this is some kind of 'catch all (of a certain type)' module and thus should be after running modules for other failures with failed RUs.

I didn't notice however that the `UnidentifiedFailure` explicitly tests for `False` of other modules (which need to be run before) while the new `RuFailed` actually does not need that, everything is solved via the usefulness value.

I will update the pull request with the next free run order number.